docs/training/apis-howto/overview.rst

```diff
@@ -1,33 +1,36 @@
-###############
- Training APIs
-###############
-
-This section describes how to use the training APIs and contains the API reference information.
-
-Determined leverages specific APIs for each Deep Learning framework. In general, users convert their
-existing training code by subclassing a Trial class and implementing methods that advertise
-components of the user's model - e.g., model architecture, data loader, optimizer, learning rate
-scheduler, callbacks, etc. This is called the Trial definition and by structuring their code in this
+#####################
+ Training API Guides
+#####################
+
+Determined leverages specific training APIs for each Deep Learning framework. The Training API
+guides describe how to use these training APIs. Each API guide contains a link to its corresponding
+API reference.
+
+With the Core API, you can plug in your existing training code. With the Trial APIs, you first
+convert your existing training code by subclassing a Trial class and implementing methods that
+define each component of training - e.g., model architecture, data loader, optimizer, learning rate
+scheduler, callbacks, etc. This is called the Trial definition. With the code structured in this
```

Comment:
> The Training API guides describe how to use these training APIs. Each API guide contains a link to its corresponding API reference.

Ooh, I like how clear this part is!

> Determined leverages specific training APIs for each Deep Learning framework ... With the Core API ... With the Trial APIs

This sentence structure suggests a taxonomy that doesn't reflect the actual taxonomy of our code.  The opening sentence, "Determined leverages specific training APIs for each Deep Learning framework" is really talking about Trial APIs.  Each trial is specific to a single deep learning framework.  In contrast, Core API is totally framework agnostic.  A sentence structure that might be more suitable might be like:

> Almost any deep learning model can be trained in Determined...  The Core API offers a flexible, low-level API for training models based in any framework...  The Trial APIs offer higher-level integrations with several specific popular frameworks...

Reply:
resolved with commit